TestCases/nonequilibrium/thermalbath/finitechemistry/weakly_ionized.cfg

```diff
@@ -0,0 +1,92 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: 2D 5x5 Square Test Case representing a thermal bath        %
+%                    with thermal nonequilibrium and finite-rate chemistry     %
+%                                                                              %
+%                   This is only serves as a validation case.                  %
```

Comment:
```suggestion
%                   This only serves as a regression case.                     %
```

Reply:
You'd think after 28 years Id be able to write properly.....